formats/developers/ome-units.txt

```diff
@@ -0,0 +1,253 @@
+The OME System of Units
+=======================
+
+.. topic:: Overview
+
+   This sections explains the OME system of Units.
```

Comment:
I'm looking at what we could add and thinking the content is pretty clear from the headings in the sidebar, let's just remove it altogether.